src/pages/AdminPage/Admin.tsx

```diff
@@ -77,7 +78,7 @@ function Admin() {
   React.useEffect(() => {
     const fetchjobs = async () => {
       try {
-        const res = await axios.get<Job[]>(`${REACT_BACKEND_ROUTE}/v1/admin`);
+        const res = await axios.get<Job[]>(`${REACT_BACKEND_ROUTE}v1/admin`);
```

Comment:
'/ ' is missing in `${REACT_BACKEND_ROUTE}v1/admin` we can use 
`${REACT_BACKEND_URL}/v1/admin` instead